pinot-common/src/main/java/org/apache/pinot/common/utils/FileUploadDownloadClient.java

```diff
@@ -89,12 +91,14 @@ public static FileUploadType getDefaultUploadType() {
 
   public static final int DEFAULT_SOCKET_TIMEOUT_MS = 600 * 1000; // 10 minutes
   public static final int GET_REQUEST_SOCKET_TIMEOUT_MS = 5 * 1000; // 5 seconds
+  public static final int DELETE_REQUEST_SOCKET_TIMEOUT_MS = 10 * 1000; // 10 seconds
 
   private static final String HTTP = "http";
   private static final String HTTPS = "https";
   private static final String SCHEMA_PATH = "/schemas";
   private static final String OLD_SEGMENT_PATH = "/segments";
   private static final String SEGMENT_PATH = "/v2/segments";
+  private static final String DELETE_SEGMENT_PATH = "/segments";
```

Comment:
Can we call it `SEGMENT_PATH` instead of `DELETE_SEGMENT_PATH`? Or why not just use `OLD_SEGMENT_PATH`?

Reply:
I wanted it to be clear that we are calling the delete API

Reply:
I think it's unnecessary, since you've already specify the behavior in the method name `getDeleteSegmentHttpUri`. Not a good idea to add more and more constants here.

Reply:
changed